Userland/Libraries/LibWeb/Worker/WebWorkerClient.h

```diff
@@ -21,6 +21,12 @@ class WebWorkerClient final
 public:
     explicit WebWorkerClient(NonnullOwnPtr<Core::LocalSocket>);
 
+    struct SocketPair {
+        int socket { -1 };
+        int fd_passing_socket { -1 };
+    };
+    SocketPair dup_sockets();
+
```

Comment:
This feels particularly jank, but refactoring the launch_server_process helper to just gimme da sockets felt out of scope for the main idea